src/main.ts

```diff
@@ -141,7 +142,26 @@ function serve(config: webpack.Configuration, args: any): Promise<void> {
 
 	const app = express();
 
-	app.use(history());
+	app.use(
+		history({
+			rewrites: [
+				{
+					from: /^.*\.(?!html).*$/,
+					to: (context: any) => {
```

Comment:
This is `any` because `request` was added in `1.5.0` but the types are still at `1.3.1`